OneSignalExample/Assets/OneSignal/Attribution/OneSignal.UnityPackage.Attribution.asmdef

```diff
@@ -16,7 +16,7 @@
     "versionDefines": [
         {
             "name": "com.onesignal.unity.core",
-            "expression": "3.0.0-beta.1",
+            "expression": "3.0.0",
```

Comment:
should this be updated to `5.0.0-beta.1`?

Reply:
I think so. Going to update other files that also have `"expression": 3.0.0`

Reply:
I have to do this post release